site.yml

```
@@ -74,15 +74,15 @@ asciidoc:
     owncloud-changelog-url: https://owncloud.com/changelog/server/
     php-supported-versions-url: https://secure.php.net/supported-versions.php
     http-status-codes-base-url: https://developer.mozilla.org/en-US/docs/Web/Status
-    minimum-php-version: 7.2
-    minimum-php-version-short-code: 72
+    minimum-php-version: 7.2.5
+    minimum-php-version-short-code: 725
     recommended-php-version: 7.4
     recommended-php-version-short-code: 74
     std-port-http: 8080
     std-port-memcache: 11211
     std-port-mysql: 3306
     std-port-redis: 6379
-    supported-php-versions: '7.2, 7.3, and 7.4'
+    supported-php-versions: '7.2.5, 7.3, and 7.4'
```

```suggestion
    supported-php-versions: '7.2.5+, 7.3, and 7.4'
```
How should we refer to this?
Any version 7.2.5 7.2.6...7.2.x should work, and we actually prefer that sites use the latest patch release.